docs/locale/fr/LC_MESSAGES/userguide.po

```diff
@@ -7,24 +7,22 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Parsec 1.7.2+dev\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-04-27 15:47+0200\n"
+"POT-Creation-Date: 2022-10-12 09:42+0200\n"
 "PO-Revision-Date: 2020-02-27 17:57+0100\n"
 "Last-Translator: \n"
 "Language: fr\n"
 "Language-Team: \n"
-"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Plural-Forms: nplurals=2; plural=(n > 1)\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Generated-By: Babel 2.10.1\n"
+"Generated-By: Babel 2.6.0\n"
```

Comment:
Is the rollback on the generator version expected ?

Reply:
Not really, sphinx generated the changes but didn't use my local Babel version (2.10.3). I'll check how to fix it